static-site/content/featured-analyses.yaml

```diff
@@ -21,7 +26,7 @@
   url: /avian-flu/h5n1/ha/2y
 
 - name: Mpox
-  description: Evolution and spread mpox virus
+  description: Evolution and spread of mpox virus (clade-IIb)
```

Comment:
slightly pedantic but this description is a bit misleading because the dropdown options in this build also include clade I as well as all-clade views

Not sure what the best solution would be. Maybe "Mpox DRC Clade-I" as the name for the new one?

Reply:
Good call - have dropped the " (clade-IIb)" and changed the new tile's title